doc/source/notebooks/intro.ipynb

```diff
@@ -11,76 +11,76 @@
+    "  - [How GPflow relates to TensorFlow: tips and tricks](tips_and_tricks.ipynb)\n",
     "\n",
     "    \n",
-    "## Advanced Needs\n",
-    "\n",
-    "GPflow also allows for more complex features and models:\n",
-    "\n",
-    "*Models:*\n",
-    "  - [MCMC](advanced/mcmc.ipynb) instead of variational inference: Hamiltonian Monte Carlo to sample the posterior GP and hyperparameters.\n",
-    "  - [Ordinal regression](advanced/ordinal_regression.ipynb) shows how to use GPflow for dealing with ordinal variables.\n",
-    "  - [Varying noise for different data points](advanced/varying_noise.ipynb) using a custom likelihood and using the SwitchedLikelihood.\n",
-    "  - [Multi-class classification](advanced/multiclass_classification.ipynb): classification in the non-binary case.\n",
-    "  - [Multi-output with coregionalisation](advanced/coregionalisation.ipynb), when not all outputs are observed at every data point.\n",
-    "  - [Multi-outputs with SVGPs](advanced/multioutput.ipynb), more efficient when all outputs are observed at all data points.\n",
-    "  - When you're dealing with a large datasets (over 1k), you want to resort to Sparse methods. In [GPs for big data](advanced/gps_for_big_data.ipynb) we show how to use GPflow's Sparse Variational GP (SVGP) model (Hensman et al., 2013; 2015)\n",
-    "  - [Manipulating kernels](advanced/kernels.ipynb) shows what covariances are included in the library, and how they can be combined to create new ones.\n",
-    "\n",
-    "*Features:*\n",
-    "  - [Natural gradients](advanced/natural_gradients.ipynb) for optimising the variational approximate posterior's parameters.\n",
+    "## Advanced needs\n",
+    "\n",
+    "This section explains the more complex models and features that are available in GPflow.\n",
+    "\n",
+    "### Models\n",
+    "  - [Markov Chain Monte Carlo (MCMC)](advanced/mcmc.ipynb): using Hamiltonian Monte Carlo to sample the posterior GP and hyperparameters.\n",
+    "  - [Ordinal regression](advanced/ordinal_regression.ipynb): using GPflow to deal with ordinal variables.\n",
+    "  - [Gaussian process regression with varying output noise](advanced/varying_noise.ipynb) for different datapoints, using a custom likelihood or the `SwitchedLikelihood`.\n",
+    "  - [Multiclass classification](advanced/multiclass_classification.ipynb) for non-binary examples.\n",
+    "  - [Multi-output models with coregionalisation](advanced/coregionalisation.ipynb): for when not all outputs are observed at every data point.\n",
+    "  - [Multi-output models with Sparse Variational Gaussian Processes (SVGPs)](advanced/multioutput.ipynb): more efficient when all outputs are observed at all data points.\n",
+    "  - [GPs for big data](advanced/gps_for_big_data.ipynb): using GPflow's SVGP model (Hensman et al., 2013; 2015). Use sparse methods when dealing with large datasets (over 1k).\n",
```

Comment:
Conceptually would probably make more sense to move this one up (before any of the multi-output ones).